sidekiq/datadog_checks/sidekiq/__init__.py

```diff
@@ -0,0 +1,7 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from .__about__ import __version__
+from .sidekiq import SidekiqCheck
```

Comment:
This module (and the associated class) doesn't seem to exist, do they?

Actually, do we even need the Python package for tile-only integrations? (Not sure on this one, might be worth comparing with an existing tile-only integration)

Reply:
True, I did not see this! And Sidekiq has a logs component so it needs the python files

Reply:
This was a typo in the logs template. I created a PR https://github.com/DataDog/integrations-core/pull/6106 to fix it

Reply:
> Sidekiq has a logs component so it needs the python files

I remember from https://github.com/DataDog/integrations-extras/pull/529 that integrations that only have logs and a tile don't need Python code: has it changed? cc @AlexandreYang 

> Looks like `contrastsecurity` is only needed for logs and assets.
>
> Hence, it does not need code related to python integration.

Reply:
^Forget this, looks like we need the Python code to ship config files, okay! :-)